src/tryceratops/settings.py

```diff
@@ -7,7 +7,11 @@
         "default": {
             "format": "%(asctime)s:%(name)s:%(process)d:%(lineno)d " "%(levelname)s %(message)s",
             "datefmt": "%Y-%m-%d %H:%M:%S",
-        }
+        },
+        "simple": {
+            "format": "%(asctime)s:%(lineno)d " "%(levelname)s %(message)s",
```

Comment:
Nice, do you think timestamp and `lineno` is good? Have you tested it?

Reply:
I didn't do a formal test but I do have code that causes the exception. When the exception hits message to stdout can probably be better. I'll make a few changes

Reply:
For the package I tested, I get this:

```
2021-07-25 07:29:52:56 DEBUG Failed to parse file /Users/ryan/PycharmProjects/tatisjr/venv/lib/python3.9/site-packages/pony/converting.py, skipping it
```

Do you think that's enough to display? 

Using the `default` formatted gets this:

```
2021-07-25 07:32:09:tryceratops.files:78830:56 DEBUG Failed to parse file /Users/ryan/PycharmProjects/tatisjr/venv/lib/python3.9/site-packages/pony/converting.py, skipping it
```

Which I think is more that is needed but ...  🤷

Reply:
I agree with you. The first one is better!

Reply:
Actually, what do you think about removing the timestamp from the first one? Do you think it's relevant?

```suggestion
            "format": "%(levelname)s %(message)s",
```

Reply:
I like this a lot. Comes out very clean and just gives the most pertinent information:

```
DEBUG Failed to parse file /Users/ryan/PycharmProjects/tatisjr/venv/lib/python3.9/site-packages/pony/converting.py, skipping it
```